src/components/Comment/CommentActions.js

```diff
@@ -122,7 +122,7 @@ export const CommentActions = ({
   const collapsable = collapsed !== undefined
   const collapseLabel = t(`styleguide/CommentActions/${ collapsed ? 'expand' : 'collapse'}`)
   return (
-    <div {...styles.root} {...(collapsable && !highlighted ? styles.collapsable : undefined)}>
+    <div {...styles.root} {...(collapsable && collapsed && !highlighted ? styles.collapsed : undefined)}>
```

Comment:
I think `{...(collapsable && collapsed && !highlighted && styles.collapsed)}`would work as well

Reply:
I thought spreading `...(false)` is nor supported, while `...(undefined)` is?